script-templates/Splunk.OTel.DotNet.psm1.template

```diff
@@ -627,16 +637,18 @@ function Get-OpenTelemetryInstallDirectory() {
 
 <#
     .SYNOPSIS
-    Gets Splunk Distribution of OpenTelemetry .NET install version. 
+    Gets the version of the `OpenTelemetry .NET AutoInstrumentation` installed with the `Splunk Distribution of OpenTelemetry .NET`. 
 #>
-function Get-OpenTelemetryInstallVersion() {
-    $installDir = Get-OpenTelemetryInstallDirectory
-
-    if ($installDir) {
-        $mainDllPath = [IO.Path]::Combine($installDir, 'net', 'OpenTelemetry.AutoInstrumentation.dll')
+function Get-OpenTelemetryAutoInstrumentationVersion() {
```

Comment:
This file is adjusted copy from the upstream https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/8a60e68da601098c401b233ea7b518274e378d47/script-templates/OpenTelemetry.DotNet.Auto.psm1.template

If possible, we should keep all methods with the same names, just provide superset of functionalities.

I ma fine with adding new method/fixing old one if needed..

Reply:
I wasn't aware of that, good point. will update keeping in line with upstream.

Reply:
Returned the name to the same as upstream and also copied the respective documentation that better describes what the cmdlet actually does.